tests/Medidata.ZipkinTracer.Core.Test/ZipkinClientTests.cs

```diff
@@ -569,6 +569,48 @@ public void RecordBinary_IsTraceOnIsFalse_DoesNotAddBinaryAnnotation()
             Assert.AreEqual(0, testSpan.Binary_annotations.Count, "There are annotations but the trace is off.");
         }
 
+        [TestMethod]
+        [TestCategory("TraceRecordTests")]
+        public void RecordLocalComponent_WithNotNullValue_AddsLocalComponentAnnotation()
+        {
+            // Arrange
+            var testValue = "Some Value";
+            var tracerClient = SetupZipkinClient();
+            spanTracerStub = MockRepository.GenerateStub<SpanTracer>(spanCollectorStub, MockRepository.GenerateStub<ServiceEndpoint>(), new List<string>(), fixture.Create<string>(), fixture.Create<string>());
```

Comment:
Here we have BIG monitors :wink:

More seriously you are right, although we don't have any established guideline for formatting (yet!), I also would recommend some limit (more like 160 chars instead of 120) and it seems that we are using the same method and same parameters to create a stub for 17 times, so I will just refactor this to a private method.

Thanks for the suggestion!